src/EventStore.Core.Tests/Http/PersistentSubscription/nack.cs

```diff
@@ -29,8 +22,10 @@ class when_nacking_a_message<TLogFormat, TStreamId> : with_subscription_having_e
 				ContentType.CompetingJson,
 				_admin);
 			Assert.AreEqual(HttpStatusCode.OK, _lastResponse.StatusCode);
-			_nackLink = json["entries"].Children().First()["links"].Children()
-				.First(x => x.Value<string>("relation") == "nack").Value<string>("uri");
+			Assert.DoesNotThrow(() => {
```

Comment:
Do the DoesNotThrow calls change the behaviour here?

Reply:
It prevents the the test host from being aborted caused by an empty collection.